client/src/components/Header.js

```diff
@@ -247,6 +249,7 @@ export default class HeaderComponent extends Component {
                 
             </>
             }
+            <FootyPoints points={this.props.footyPoints}/>
```

Comment:
I'm thinking whether we merge the ambassador portal in first and have the FootyPoints badge live in the welcomeBadge component?  Otherwise we have 3 components in the first column of the header which I can get with. Your call!

Reply:
![image](https://user-images.githubusercontent.com/31665569/107880269-c0521580-6eab-11eb-8276-bbc23bc7f38f.png)
Let's add some styling and be deliberate with stacking behavior.
Let's add `style = {{ margin: 5px }}` or better yet let's always get these component to stack vertically. Use a `Grid.Row`for accessContext dropdown and the badge

Reply:
Hm interesting, and yeah I'm indifferent on which to merge in first, sounds like it should get moved to the welcomeBadge component, but I think it should just be a one line thing to get shifted over? And the supporting test should still work, so doesn't seem like a big deal to me either way

Reply:
Also yeah I'll look into the styling, good call. I was doing it without that dropdown present so it wasn't quite so obvious that it was out of line